README.md

```diff
@@ -1,7 +1,8 @@
 # BiggerQuery &mdash; Python library for BigQuery
 
 BiggerQuery is a Python library which simplifies working with BigQuery datasets. It wraps BigQuery client, providing elegant
- API for most common use cases.
+ API for most common use cases. It also provides API for creating dataflow pipelines and reading/writing
```

Comment:
Check the whole readme using grammarly please, there are multiple places missing "the", "a", "an".

Reply:
Done